engines/osxnative/resultset.go

```diff
@@ -97,3 +96,18 @@ func (r resultset) ExtractFolder(path string, handler engines.FileHandler) error
 		return nil
 	})
 }
+
+func (r resultset) Dispose() error {
+	err := r.taskUser.delete()
+	if err != nil {
+		r.context.LogError("Error removing user \""+r.taskUser.name+"\": ", err)
+		exitError, ok := err.(*exec.ExitError)
+		if ok {
+			r.context.LogError(string(exitError.Stderr))
```

Comment:
the task context log is closed before the Dispose() method is called on a resultset.